SalesforceSDK/Salesforce.SDK.Hybrid.SmartStore/Source/Store/SmartStore.cs

```diff
@@ -232,5 +238,75 @@ public static string GetSoupTableName(long soupId)
             return SDK.SmartStore.Store.SmartStore.GetSoupTableName(soupId);
         }
 
+        /// <summary>
+        /// Native implementation of pgCloseCursor
+        /// </summary>
+        /// <param name="id"></param>
+        public void CloseCursor(int id)
+        {
+            if (_cursors.ContainsKey(id))
+            {
+                _cursors.Remove(id);
+            }
+        }
+
+        /// <summary>
+        /// Native implementation of pgMoveCursorToPageIndex
+        /// </summary>
+        public string MoveCursorToPageIndex(int id, int index)
+        {
+            if (!_cursors.ContainsKey(id))
```

Comment:
@kchitalia should I be logging anything in these functions if something unexpected happens?